File: PizzaBackendServer/WebApi/BLL/Models/PizzaModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DAL.Data;

namespace BLL.Models
{
    public class PizzaModel
    {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public decimal Price { get; set; }
        public string Description { get; set; } = null!;
        public bool Prescence { get; set; }
        public string Consistance { get; set; } = null!;
        public int SizeId { get; set; }
        public string? Photo { get; set; }
        public PizzaModel() { } 

        public PizzaModel(Pizza p)
        {
            Id = p.Id;
            Name = p.Name;
            Price = p.Price;
            Description = p.Description;
            Prescence = !(p.Prescence==0);
            Consistance = p.Consistance;
            SizeId = p.SizeId;
            Photo = p.Photo;
        }
    }
}

File: PizzaMobileApp/pizzer_mobile/lib/models/pizza_model.dart
class PizzaModel {
  int? id;
  String? name;
  double? price;
  String? description;
  bool? prescence;
  String? consistance;
  int? sizeId;
  String? sizeName;
  String? photo;

  PizzaModel(
      {this.id,
      this.name,
      this.price,
      this.description,
      this.prescence,
      this.consistance,
      this.sizeId,
      this.sizeName,
      this.photo});

  PizzaModel.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    name = json['name'];
    price = json['price'];
    description = json['description'];
    prescence = json['prescence'];
    consistance = json['consistance'];
    sizeId = json['sizeId'];
    sizeName = json['sizeName'];
    photo = json['photo'];
  }
}
